mr/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -163,11 +158,6 @@ public ConfigBuilder preferLocality() {
       return this;
     }
 
-    public ConfigBuilder useHiveRows() {
```

Comment:
Shouldn't these be deprecated for a release instead of removed? Keeping them won't prevent us from removing other Hive classes.

Reply:
Actually, I don't think this matters. I'm good with removing it.